libbeat/docs/dashboards.asciidoc

```diff
@@ -6,7 +6,14 @@ the perfect dashboards for your needs. But this flexibility can be a bit
 overwhelming at the beginning, so we have created some sample dashboards to get you
 started and to demonstrate what is possible based on the Beats data.
 
-NOTE: The Beats dashboards require Kibana 4 or later.
+NOTE: The Beats dashboards require Kibana 5 or later.
```

Comment:
In fact the Beats dashboards work with both Kibana versions, but Kibana 5.0 works only with Elasticsearch 5.0 (needs to be confirmed by Kibana team). At a second thought, not sure if we should include this information in our dashboards. Maybe we should just avoid this information for now as it's not related to Beats.